installer/scripts/npm_startup.sh

```diff
@@ -72,3 +74,7 @@ else
         echo "File $release_file not found!"
 fi
 
+if [ $require_notify -eq 1 ];
+then
+    echo "Please configure your firewall (if running), to allow connections for destination port 8084 over TCP for NPM solution to run"
```

Comment:
Ok, I see that it doesn't change in the current implementation. Could you just clean it up by printing $tcp_port instead of 8084?